CoreScriptsRoot/Modules/Panel3D.lua

```diff
@@ -85,13 +88,20 @@ function Panel3D.Get(panelId)
 
 		panel.transparencyCallbacks = {}
 
+		panel.OnMouseEnter = false
+		panel.OnMouseLeave = false
```

Comment:
It's probably not worth it at this point, and saving lines of code isn't a super high priority after all!